test/test_imu_downlink/test_imu_downlink.cpp

```diff
@@ -132,6 +166,7 @@ int test_rockblock_report_monitor()
     RUN_TEST(test_imu_downlink_mandburn);
     RUN_TEST(test_imu_downlink_regburn);
     RUN_TEST(test_imu_downlink_deployed);
+    //RUN_TEST(test_imu_downlink_delay); // Delays a minute, uncomment this out to test the time control logic.
```

Comment:
I would probably uncomment this. Its worthwhile to keep in, especially since it can be skipped if needed